test/test_helper.rb

```diff
@@ -47,9 +47,8 @@ def serialization_options(options)
 
 require 'support/rails5_shims'
 
-require 'fixtures/active_record'
-
-require 'fixtures/poro'
+require 'support/fixtures/active_record'
+require 'support/fixtures/poro'
```

Comment:
yeah. 

I'm kinda thinking tests should utilize less of the poros, and just define what they need in the test class. But... for the future, I guess.